plural_rules.go

```diff
@@ -195,7 +196,6 @@ var singularToPluralSuffixList = []singularToPluralSuffix{
 	{"oose", "eese"},
 	{"ouse", "ouses"},
 	{"ovum", "ova"},
-	{"rion", "ria"},
```

Comment:
it also not always correct, or it seems like the case of "~rions" is more popular.